bottery/exceptions.py

```diff
@@ -9,3 +9,13 @@ class ValidationError(Exception):
 
 class BotteryDeprecationWarning(Warning):
     pass
+
+
+class PlatformError(Exception):
+    def __init__(self, platform, message):
+        super().__init__()
+        self.platform = platform
+        self.message = message
+
+    def __str__(self):
+        return '[{}] {}'.format(self.platform, self.message)
```

Comment:
I started questioning myself if the `[{}] {}` is a pattern that we should keep following. In this case, I think we don't need the `self.platform`, the exception itself should be able to give all the info we need. Maybe `class TelegramException(PlatformException)` is a good idea. What do you think?